app/Screens/Splashscreen.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {View, Image, StyleSheet} from 'react-native';
+import {View, Image, StyleSheet, AsyncStorage} from 'react-native';
```

Comment:
Can Remove this as it is never used in Code !

Reply:
Noted, next merge in the development branch will be removed.